.github/release-drafter.yml

```
@@ -31,6 +31,7 @@ categories:
 
   - title: ":arrow_up: Dependency Updates"
     label: "dependencies"
+    collapse-after: 3
```

```suggestion
    collapse-after: 1
```
0 Can not be used, but setting 1 will use summary when there are 2 and more.